deluge/ui/console/modes/preferences/preference_panes.py

```diff
@@ -251,7 +251,7 @@ def create_pane(self, core_conf, console_config):
 
         self.add_header(_('Outgoing Interface'), space_above=True)
         self.add_text_input('outgoing_interfaces', '%s:' % _('IP address of the interface to make connections out on '
-                                                          '(leave empty for default)'),
+                                                             '(leave empty for default)'),
```

Comment:
I'd prefer this change:

```diff
-        self.add_text_input('listen_interface', '%s:' % _('IP address of the interface to listen on '
-                                                          '(leave empty for default)'),
-                            core_conf['listen_interface'])
+        self.add_text_input(
+            'listen_interface',
+            _('IP address of the interface to listen on (leave empty for default):'),
+            core_conf['listen_interface'],
+        )
```

The translation string was incorrect since the colon could not be moved by the translator and there is no need for substitution as it is a fixed string.